src/CSET/operators/collapse.py

```diff
@@ -163,10 +181,35 @@ def collapse_by_hour_of_day(
     To apply this operator successfully there must only be one time dimension.
     Should a MultiDim exception be raised the user first needs to apply the
     collapse operator to reduce the time dimensions before applying this
-    operator.
+    operator. If multi_case is true the collapse_by_lead_time operator is
+    applied and performs this step.
     """
     if method == "PERCENTILE" and additional_percent is None:
         raise ValueError("Must specify additional_percent")
+    if (
```

Comment:
```suggestion
    elif (
```
These conditions are mutually exclusive, so we only need to check them if the previous one didn't match.